aladdin.sh

```diff
@@ -123,9 +123,17 @@ function enter_minikube_env() {
 }
 
 function _extract_cluster_config_value() {
+    # Try extracting config from cluster config.json, default config.json, then aladdin config.json
     local value
     value="$1"
-    jq -r ".$value" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json"
+    output=$(jq -r ".$value" "$ALADDIN_CONFIG_DIR/$CLUSTER_CODE/config.json")
+    if [[ "$output" == null ]]; then
```

Comment:
This seemed verbose, so I spent some time digging around in the jq man page because I was curious if there was another way to say this / to remove the actual 'null' value which feels awkward in a shell script.

So, it looks like we could do, e.g.,
```
# Find the result from the first input file that matches .$value
jq -enr "first(inputs | .$value // empty)" "$config1" "$config2" "$config3"
```
And that would echo the first match or nothing if nothing matched, and also return nonzero if there were no matches (so we could distinguish that case from empty string).

That said, now that I look at it, that might be more confusing for anyone who finds it in the future, and it's not more correct.

Well, now I know more about jq :man_shrugging:

Reply:
Actually I take that back.  As it is right now, with the `-r` option, the string "null" is indistinguishable from an actual null.  The code I specified above is more correct.

Reply:
EDIT: Updated the initial command to add quotes around the selector, to allow for keys with spaces in them

Reply:
awesome.. this is way better